webrtc.html

```diff
@@ -2437,10 +2449,13 @@ <h4>
                               <li>
                                 <p>
                                   If a suitable transceiver was found
-                                  (<var>transceiver</var> is set) and
-                                  <var>sendEncodings</var> is non-empty, set
+                                  (<var>transceiver</var> is set),
+                                  <var>proposedSendEncodings</var> is non-empty,
+                                  and
+                                  <var>transceiver</var>.{{RTCRtpTransceiver/[[Sender]]}}.{{RTCRtpSender/[[SendEncodings]]}}
+                                  is empty, then set
```

Comment:
This line is bogus since [[[SendEncodings](https://w3c.github.io/webrtc-pc/#dfn-sendencodings)]] is never empty.